ops/terraform/services/server/modules/bfd_server_dashboards/templates/bfd-dashboard.tftpl

```diff
@@ -547,11 +540,222 @@
+                            "yAxis": "right"
+                        },
+                        {
+                            "color": "#b088f5",
+                            "label": "6 - 9 instances",
+                            "value": 400000000,
+                            "yAxis": "right"
+                        }
+                    ]
+                },
+                "liveData": true
+            }
+        },
+        {
+            "height": 5,
+            "width": 12,
+            "y": 49,
+            "x": 0,
+            "type": "metric",
+            "properties": {
+                "metrics": [
+                    [ "AWS/ELB", "HealthyHostCount", "LoadBalancerName", "bfd-${env}-fhir" ],
+                    [ "AWS/AutoScaling", "GroupDesiredCapacity", "AutoScalingGroupName", "bfd-${env}-fhir-375", { "yAxis": "right", "visible": false } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "${region}",
+                "stat": "Average",
+                "period": 60,
+                "liveData": true,
+                "yAxis": {
+                    "left": {
+                        "min": 0,
+                        "max": 1.2
+                    }
+                },
+                "annotations": {
+                    "horizontal": [
+                        {
+                            "label": "Alert Threshold",
+                            "value": 1
+                        }
+                    ]
+                },
+                "title": "Average of Healthy Hosts in bfd-${env}-fhir Elastic Load Balancer"
+            }
+        },
+        {
+            "height": 7,
+            "width": 6,
+            "y": 7,
+            "x": 18,
+            "type": "metric",
+            "properties": {
+                "metrics": [
+                    [ { "expression": "SEARCH('http-requests/count/non-2xx-responses ${namespace} client_ssl', 'Sum', 60)", "label": "non-2xx responses", "id": "e1" } ],
+                    [ { "expression": "SEARCH('http-requests/count/500-responses ${namespace} client_ssl', 'Sum', 60)", "label": "500-responses", "id": "e2" } ],
+                    [ "${namespace}", "http-requests/count/500-responses", "client_ssl", "${bb_client_ssl}", { "region": "${region}", "visible": false, "id": "m1" } ],
+                    [ "...", "${ab2d_client_ssl}", { "region": "${region}", "visible": false, "id": "m2" } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "${region}",
+                "stat": "Sum",
+                "period": 60,
+                "liveData": true,
+                "yAxis": {
+                    "left": {
+                        "showUnits": false,
+                        "label": "Response Count"
+                    }
+                },
+                "title": "Non-2xx and 500 Response Count Per-Partner"
```

Comment:
Hey @malessi This is more of a question than anything. Are we only tracking 500 Response codes or are there other 5XX we track?

Reply:
We log all response codes for all requests. But summarize these to non-200's in dashboards.